docs/api/navigators/StackNavigator.md

```diff
@@ -118,7 +118,7 @@ All `navigationOptions` for the `StackNavigator`:
 
 - `title` - a title (string) of the scene
 - `header` - a config object for the header bar:
-  - `visible` - Boolean toggle of header visibility. Only works when `headerMode` is `screen`.
+  - `bar` - React Element or a function that given `HeaderProps` returns a React Element, to display as a header. Setting to `null` hides header.
```

Comment:
Shouldn't this say 'Component' instead of 'Element'? Related to my confusion below